code/__DEFINES/misc.dm

```diff
@@ -453,9 +453,6 @@ GLOBAL_LIST_INIT(pda_styles, list(MONO, VT, ORBITRON, SHARE))
 // Used by PDA and cartridge code to reduce repetitiveness of spritesheets
 #define PDAIMG(what) {"<span class="pda16x16 [#what]"></span>"}
 
-//Filters
-#define AMBIENT_OCCLUSION filter(type="drop_shadow", x=0, y=-2, size=4, border=4, color="#04080FAA")
```

Comment:
Why remove the define? The define makes it easier to find and keep track of a specific filter's properties.

Reply:
Because a filter with these specific properties is only used once, and with the `add_filter` proc you don't even use the in-built `filter()` proc, so it'd just be a list of properties. 

I can use defines if you really want, but I don't think there's much point.